git/repo/base.py

```diff
@@ -368,41 +403,45 @@ def submodule_update(self, *args, **kwargs):
     #}END submodules
 
     @property
-    def tags(self):
+    def tags(self) -> 'IterableList':
         """A list of ``Tag`` objects that are available in this repo
         :return: ``git.IterableList(TagReference, ...)`` """
         return TagReference.list_items(self)
 
-    def tag(self, path):
+    def tag(self, path: PathLike) -> TagReference:
         """:return: TagReference Object, reference pointing to a Commit or Tag
         :param path: path to the tag reference, i.e. 0.1.5 or tags/0.1.5 """
         return TagReference(self, path)
 
-    def create_head(self, path, commit='HEAD', force=False, logmsg=None):
+    def create_head(self, path: PathLike, commit: str = 'HEAD',
+                    force: bool = False, logmsg: Optional[str] = None
+                    ) -> 'SymbolicReference':
         """Create a new head within the repository.
         For more documentation, please see the Head.create method.
 
         :return: newly created Head Reference"""
         return Head.create(self, path, commit, force, logmsg)
 
-    def delete_head(self, *heads, **kwargs):
+    def delete_head(self, *heads: HEAD, **kwargs: Any) -> None:
```

Comment:
I think `HEAD` is too specific - any symbolic reference can be created that way.

Reply:
amended in #1202